.env.example

```diff
@@ -28,4 +28,4 @@ NODE_HOSTNAME=localhost
 NODE_PORT=3001
 MATOMO_ADD_USER_GOAL_ID=3
 MATOMO_REMOVE_USER_GOAL_ID=4
-COOKIE_SECURE_ONLY=false
\ No newline at end of file
+ENV_NAME=local
```

Comment:
@kwamenyantakyi-ch remember to add this variable to `services/modules/acsp-manage-users/acsp-manage-users-web.docker-compose.yaml` in docker-chs-development repository